src/mrpro/operators/_FourierOp.py

```diff
@@ -173,7 +179,7 @@ def get_target_pattern(fft_dims: tuple[int, ...], nufft_dims: tuple[int, ...], i
             elif dim in ignore_dims:
                 ignore_dims_str += ' ' + dim_str
 
-        target_pattern = f'(other{fft_dims_str}{ignore_dims_str}) coil{nufft_dims_str}'
+        target_pattern = f'(other{fft_dims_str}{ignore_dims_str}) coils{nufft_dims_str}'
```

Comment:
If you change coil -> coils here, you also have to change it when this pattern is used.
Also, maybe  make other capture multiple dimensions already via ... ?

Reply:
example would be line 230